scripts/adlsgen2setup.ps1

```diff
@@ -0,0 +1,109 @@
+## Set the preference to stop on the first error
+$ErrorActionPreference = "Stop"
```

Comment:
Can you write a sh equivalent? Or alternatively write most of this using the Azure Python SDK so that the sh/ps1 files are just small wrappers?

Reply:
sure, i can take a look at doing this

Reply:
writing in python, writing sh file